src/osd/OSD.cc

```diff
@@ -1660,6 +1660,13 @@ bool OSD::asok_command(string command, cmdmap_t& cmdmap, string format,
     } else {
       op_tracker.dump_ops_in_flight(f);
     }
+  } else if (command == "dump_blocked_ops") {
+    if (!op_tracker.tracking_enabled) {
+      ss << "op_tracker tracking is not enabled";
```

Comment:
@gregsfortytwo . if tracking disable, monitor don't know ops whether blocked. Your concern is right. We need print blocked ops whether tracking enable or disable. But now tracker can't do this.  Backed to #5168, why did i propose this?  In maillist, someone said the overhead of optracker.   Reduce overhead and don't disable optracker is the best option. But i don't know how to do this currently.